Casks/i/ia-presenter.rb

```diff
@@ -0,0 +1,25 @@
+cask "ia-presenter" do
+  version "1.1.2-11002"
+  sha256 "ce026bfc2155104e68e5eeb9eaa29d0a279e7c5de7771ea55e19ff5a4ea2740e"
+
+  url "https://files.ia.net/presenter/release/iA-Presenter-#{version}.zip"
+  name "ia-presenter"
+  desc "Create presentation slides from a Markdown document"
+  homepage "https://ia.net/presenter"
+
+  livecheck do
+    url "https://iapresenter.s3.amazonaws.com/release/updates.xml"
+    strategy :sparkle do |item|
+      "#{item.short_version}-#{item.version}"
+    end
+  end
+
+  depends_on macos: ">= :monterey"
```

Comment:
Upstream defines it as:
```suggestion
  depends_on macos: ">= : big_sur"
```

Reply:
Yup, already changed it. Not sure if it defines the correct version tho? It needs 11.3, not 11.0, I didn't see a way to put 11.3 in the cask cookbook so I just put monterey.

![CleanShot 2024-01-06 at 16 30 09@2x](https://github.com/Homebrew/homebrew-cask/assets/62390580/6c70c4db-d1b9-406f-8f24-c48c0209aed9)